src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -362,21 +363,31 @@ def consRecordAge(self, rse):
         :return:    rse or raises MSUnmergedPlineExit
         """
         rseName = rse['name']
+
+        if rseName not in self.rseConsStats:
+            msg = "RSE: %s Missing in stats records at Rucio Consistency Monitor. " % rseName
+            msg += "Skipping it in the current run."
+            self.logger.info(msg)
```

Comment:
Shouldn't this be set to warning level? If you worry that testbed will be full of such warning records, I'd rather have it that way then overlooking INFO records in production.

Reply:
agreed